src/luminator.ts

```diff
@@ -82,14 +84,14 @@ class Luminator {
   /**
    * @description Return status if it is a valid AxiosError
    * @param {AxiosError} error
-   * @throws {Error} non axios error
+   * @throws {LuminatorError} non axios error
    * @return {number}
    */
   private static getStatusFromAxiosError(error: AxiosError): number {
     try {
       return error.response.status;
     } catch (e) {
-      throw error;
+      throw new LuminatorError(`NON_AXIOS_ERROR`);
```

Comment:
Not sure that really useful to use uppercase message.

Reply:
its maybe better to :

```
throw new LuminatorError(e);
```

Reply:
The original error about the line above "cannot use property status of undefined". I don't think this a relevant error, is it ?

Reply:
It means the request has no status, just a proxy error

Reply:
If it works for our needed, it's okay for me.